crates/bevy_diagnostic/src/log_diagnostics_plugin.rs

```diff
@@ -2,103 +2,102 @@ use super::{Diagnostic, DiagnosticId, Diagnostics};
 use bevy_app::prelude::*;
 use bevy_core::{Time, Timer};
 use bevy_ecs::{IntoSystem, Res, ResMut};
+use bevy_log::{debug, info};
 use bevy_utils::Duration;
 
-/// An App Plugin that prints diagnostics to the console
-pub struct PrintDiagnosticsPlugin {
+/// An App Plugin that logs diagnostics to the console
+pub struct LogDiagnosticsPlugin {
     pub debug: bool,
     pub wait_duration: Duration,
     pub filter: Option<Vec<DiagnosticId>>,
 }
 
-/// State used by the [PrintDiagnosticsPlugin]
-pub struct PrintDiagnosticsState {
+/// State used by the [LogDiagnosticsPlugin]
+pub struct LogDiagnosticsState {
```

Comment:
This is `pub` but is not exported from the crate

Maybe use `pub(crate)`?

Reply:
It has to be `pub` because it's in a public interface:
```
crate-visible type `LogDiagnosticsState` in public interface
can't leak crate-visible type
```
It's public, just not user addressable...

Reply:
If we stop exporting the systems, then that error goes away. I'm not sure why we would be exporting the systems anyway?

It was created as public in https://github.com/bevyengine/bevy/commit/704a74266199c7ee51b8d7c71a3a6dd375c816f3, which was in May. I suspect it's just an oversight.

Reply:
`pub`s are removed